tox.ini

```diff
@@ -16,16 +15,14 @@ whitelist_externals =
     sh
     sleep
 deps =
-    dj18: -rtest_requirements/django-1.8.txt
-    dj19: -rtest_requirements/django-1.9.txt
-    dj110: -rtest_requirements/django-1.10.txt
-    dj111: -rtest_requirements/django-1.11.txt
+    -r{toxinidir}/test_requirements.txt
+    dj111: Django>=1.11,<2.0
+    dj20: Django>=2.0,<2.1
+    dj21: Django>=2.1.4,<2.2
```

Comment:
Changed to 2.1.4 because older version contains bug with urlencode:
https://code.djangoproject.com/ticket/29627